resources/functionMap.php

```diff
@@ -1644,7 +1644,7 @@
 'DateTimeZone::getTransitions' => ['list<array{ts: int, time: string, offset: int, isdst: bool, abbr: string}>', 'timestamp_begin='=>'int', 'timestamp_end='=>'int'],
 'DateTimeZone::listAbbreviations' => ['array<string, list<array{dst: bool, offset: int, timezone_id: string|null}>>'],
 'DateTimeZone::listIdentifiers' => ['list<string>', 'what='=>'int', 'country='=>'string'],
-'db2_autocommit' => ['mixed', 'connection'=>'resource', 'value='=>'int'],
+'db2_autocommit' => ['DB2_AUTOCOMMIT_OFF|DB2_AUTOCOMMIT_ON|bool', 'connection'=>'resource', 'value='=>'DB2_AUTOCOMMIT_OFF|DB2_AUTOCOMMIT_ON'],
```

Comment:
What does the function do if it receives something that isn't `\DB2_AUTOCOMMIT_OFF|\DB2_AUTOCOMMIT_ON`? Like `1337` for example?

Reply:
A warning is emitted and the function returns `false`:
```
Warning: db2_autocommit(): value must be one of DB2_AUTOCOMMIT_ON or DB2_AUTOCOMMIT_OFF
```
See https://github.com/php/pecl-database-ibm_db2/blob/4eebfe146a873e25af26fbca30f480747cd237af/ibm_db2.c#L3522-L3525.